### PR TITLE
go-runtime

### PR DESCRIPTION
**Pull request by sciabarracom**
Changes to include go-runtime in wskdeploy

### CODE REVIEW

File: runtimes/runtimes_test.go [hunk 1]
```diff
@@ -30,9 +31,12 @@ func TestParseOpenWhisk(t *testing.T) {
 	converted := ConvertToMap(openwhisk)
 	println(converted["nodejs"])
 	println(converted["python"])
+	//println(converted["go"])
 	assert.Equal(t, 2, len(converted["nodejs"]), "not expected length")
 	assert.Equal(t, 2, len(converted["php"]), "not expected length")
 	assert.Equal(t, 1, len(converted["java"]), "not expected length")
 	assert.Equal(t, 6, len(converted["python"]), "not expected length")
 	assert.Equal(t, 2, len(converted["swift"]), "not expected length")
+	// cannot pass this test until it is deployed in OpenWhisk
```
Comment (csantanapr) [inline runtimes/runtimes_test.go]: go is now part of the deployment runtimes in OprnWhisk, what do you mean?

Comment (sciabarracom) [reply runtimes/runtimes_test.go]: The test looks like it tries to parse BlueMix runtime.json, and it fails

Comment (csantanapr) [reply runtimes/runtimes_test.go]: ok understood, leave commented out

**Comment by csantanapr** [comment]
I fixed 2 problems, but now after deployment of OpenWhisk it fails on
```
==> Error: wskdeploy.go [128]: [ERROR_COMMAND_FAILED]: Command: [wskdeploy]: Error: manifestreader.go [93]: [ERROR_YAML_FILE_FORMAT_ERROR]: File: [manifest.yaml]: 
==> manifest_parser.go [617]: [ERROR_YAML_INVALID_RUNTIME]: File: [manifest.yaml]: [helloGo] has not specified any runtime and the action source file extension [go] is not supported.
==> Action [helloGo]: Runtime [NOT SPECIFIED]: Supported Runtimes [swift:3.1.1, swift:4.1, nodejs:6, nodejs:8, java, php:7.1, php:7.2, python, python:2, python:3, ruby:2.5]
```

Don't understand why, since the default deploy includes go

**Comment by csantanapr** [comment]
@sciabarracom see if you can try a deployment with a local openwhisk and see if you can reproduce

**Comment by csantanapr** [comment]
@pritidesai could take a look on why we get the error?
```
file extension [go] is not supported.
```
go extension was added

**Comment by csantanapr** [comment]
Found the problem thanks for @houshengbo  openwhisk runtimes.json didn't have go in the correct location.
Now being fixed with this PR https://github.com/apache/incubator-openwhisk/pull/4096

Once that PR is merged, we can re-run the build in Travis.

**Comment by csantanapr** [comment]
Woot!

**Comment by pritidesai** [comment]
thanks @sciabarracom for adding this support 👍

**Comment by pritidesai** [comment]
yup thanks @csantanapr, you are champ 🥇

**Pull request merged**